package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "nuxt-ts",
+    "dev": "HOST=0.0.0.0 nuxt-ts",
```

Comment:
Would it be better to put this in nuxt.config.ts?

Reply:
You're right, I didn't think to look into it. Thanks!